server/player.lua

```diff
@@ -97,6 +97,7 @@ function QBCore.Player.CheckPlayerData(source, PlayerData)
     if source then
         PlayerData.source = source
         PlayerData.license = PlayerData.license or QBCore.Functions.GetIdentifier(source, 'license')
+        PlayerData.discord = PlayerData.discord or QBCore.Functions.GetIdentifier(source, 'discord'):sub(9)
```

Comment:
Forgot to add.  What is this is null?  calling :sub on a null may cause an issue.  Requires testing.